src/gentropy/common/spark_helpers.py

```diff
@@ -433,3 +434,31 @@ def get_value_from_row(row: Row, column: str) -> Any:
     if column not in row:
         raise ValueError(f"Column {column} not found in row {row}")
     return row[column]
+
+
+def enforce_schema(
+    expected_schema: Any,
+) -> Callable[..., Any]:
+    """A function to ensure the schema of a function output follows expectation.
```

Comment:
```suggestion
    """A function to enforce the schema of a function output follows expectation.
```
Add information that the decorated function output will have:
- fields that exist in the function output, but are missing from expected schema will be dropped
- fields that are missing from function output, but are present in expected schema will be added with Null values
- fields that have incorrect data types will be automatically enforced to follow the schema data types

Reply:
Good point. Explanation added.